.github/workflows/pg-migtests.yml

```diff
@@ -43,6 +43,20 @@ jobs:
       env:
         ON_INSTALLER_ERROR_OUTPUT_LOG: Y
 
+    - name: Fetch debezium-server
+      uses: docker://amazon/aws-cli:2.7.27
+      with:
+        args: s3 cp s3://voyager-automation-data/debezium-latest.tar.gz .
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.RAHULB_S3_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.RAHULB_S3_SECRET_ACCESS_KEY }}
+        AWS_DEFAULT_REGION: "us-west-2"
+    
+    - name: Unpack debezium-server
+      run: |
+        sudo tar -xvzf debezium-latest.tar.gz -C /opt/
```

Comment:
Recently the dbzm installer script is modified to install debezium-server at `/opt/yb-voyager/`.